microshift_welcome/index.adoc

```diff
@@ -21,16 +21,15 @@ To browse the {microshift-short} {product-version} documentation, use one of the
 To get started with {microshift-short}, use the following links:
 
 //text is in main assembly for the sake of cross references
-//* xref:../microshift_getting_started/microshift-understanding.adoc#microshift-understanding[Understanding {product-title}]
-//* xref:../microshift_install/microshift-install-rpm.adoc#microshift-install-rpm[Installing {product-title}]
+* xref:../microshift_install_rpm/microshift-install-rpm.adoc#microshift-install-rpm[Installing {product-title}]
 * xref:../microshift_release_notes/microshift-4-16-release-notes.adoc#microshift-4-16-release-notes[{product-title} release notes]
 
 For related information, use the following links:
 
 * link:https://access.redhat.com/documentation/en-us/red_hat_device_edge/4/html/overview/device-edge-overview[Red Hat Device Edge overview]
-* link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/composing_installing_and_managing_rhel_for_edge_images/index[RHEL for Edge documentation]
-* link:https://docs.openshift.com/container-platform/latest/welcome/index.html[OpenShift Container Platform documentation]
-* link:https://docs.openshift.com/container-platform/latest/welcome/index.html[{OCP} documentation]
+* link:https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/composing_installing_and_managing_rhel_for_edge_images/index[RHEL for Edge documentation]
+* link:https://docs.openshift.com/container-platform/{ocp-version}/welcome/index.html[OpenShift Container Platform documentation]
+* link:https://docs.openshift.com/container-platform/{ocp-version}/welcome/index.html[{OCP} documentation]
 
 include::modules/microshift-about.adoc[leveloffset=+1]
```

Comment:
This module isn't included in this PR, but in the preview, I noticed that a list isn't being presented correctly:
<img width="782" height="342" alt="Screenshot 2025-09-16 at 11 11 15 AM" src="https://github.com/user-attachments/assets/e5d77183-b5de-4a29-b56d-d138428a2bb9" />